Source/WebKit/UIProcess/Cocoa/ExtensionCapabilityGranter.mm

```diff
@@ -127,7 +127,13 @@ static bool finalizeGrant(const String& environmentIdentifier, AuxiliaryProcessP
     }
 
     if (grant.isValid()) {
-        iterator->value = WTFMove(grant);
+        auto& existingGrant = iterator->value;
+        ASSERT(!existingGrant.isValid());
+        if (existingGrant.isValid()) {
+            GRANTER_RELEASE_LOG_ERROR(environmentIdentifier, "grant not expected to be valid");
+            granter.invalidateGrants(Vector<ExtensionCapabilityGrant>::from(WTFMove(existingGrant)));
```

Comment:
Wouldn't this build?
```
granter.invalidateGrants({ WTFMove(existingGrant) });
```

Reply:
Ah, good point, I get the following error when trying this:

```
Copy constructor is implicitly deleted because 'ExtensionCapabilityGrant' has a user-declared move constructor
```